### PR TITLE
editor: UI rework step 1

### PR DESCRIPTION
**Pull request by elbotho**
- first steps toward [new ui style](https://miro.com/app/board/uXjVP0cDuaI=/?moveToWidget=3458764548825783346&cot=14)
- normalize used yellows
- use yellow where it's relatively easy

### Preview
- https://frontend-git-editor-ui-rework-serlo.vercel.app/entity/create/Course/200247
- https://frontend-git-editor-ui-rework-serlo.vercel.app/entity/repository/add-revision/1555

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -54,6 +57,10 @@ module.exports = {
           DEFAULT: colors.red[900],
           100: colors.red[100],
         },
+        sunflower: {
+          DEFAULT: '#FFBE5E',
+          300: '',
+        },
```
Comment (elbotho) [inline tailwind.config.js]: memo: check again.

Comment (Entkenntnis) [reply tailwind.config.js]: nice color name

File: src/data/en/index.ts [hunk 2]
```diff
@@ -1105,7 +1106,7 @@ Your Community-Support 💚`,
       body: `<p>Hi <b>{{ .Identity.traits.username }}</b>,</p>
 <p>We are excited to have you at serlo.org 🎉</p>
 <p>Please verify your account by clicking the following link:<br/>
-<a style="color: #007EC1 !important;" href="{{ .VerificationURL }}">{{ .VerificationURL }}</a>
+<a style="color: ${colors.brand} !important;" href="{{ .VerificationURL }}">{{ .VerificationURL }}</a>
```
Comment (Entkenntnis) [inline src/data/en/index.ts:1109]: clever usage of translation :)

**Review by Entkenntnis — APPROVED** [review]
thanks! changes look good

File: src/data/en/index.ts (hunk 2 above)
Comment (elbotho) [reply src/data/en/index.ts:1109]: uh, hack at best. but it was the simplest way to implement it.

**Comment by elbotho** [comment]
@Entkenntnis did some more color refactoring (also affecting rest of frontend), maybe give it a quick glance if you are okay with those

**Review by Entkenntnis — APPROVED** [review]
very nice visual improvement and finally more consistency 😍

**Pull request merged**